src/main/resources/application-alpha.properties

```diff
@@ -0,0 +1,2 @@
+db.migrate.dropAllOnStart=true
+sam.basePath=https://sam.dsde-alpha.broadinstitute.org
```

Comment:
These two variables will live in the alpha helm.

Reply:
Should these be moved before merging this?

Reply:
I agree that these should be committed in the PR and this file removed before we merge.

Reply:
So we will not need to have an alpha profile, right?